tests_phi/pages/dashboard_page.py

```diff
@@ -82,3 +83,10 @@ def delete_page(self, page_name):
         self.select_global_selecting_item("Delete")
         browser.get_alert().accept()
```

Comment:
You have method browser_helper, You should use from it

Reply:
done